sensing/autoware_radar_objects_adapter/src/radar_objects_adapter.cpp

```diff
@@ -263,8 +263,18 @@ void RadarObjectsAdapter::parse_as_detections(
   for (const auto & input_object : input_msg.objects) {
     autoware_perception_msgs::msg::DetectedObject output_object;
 
+    // Populate common fields
     const auto & yaw = input_object.orientation;
     populate_common_fields(input_object, output_object, yaw);
+
+    // Set flags for kinematics
+    output_object.kinematics.has_position_covariance = true;
```

Comment:
[Just a comment]
We only define array number in `RadarObject` msg, so I prefer to do sanity check in this node. (ex: if all of array number is 0, covariance is unavailable)

```msg
float32[6] position_covariance
float32[6] velocity_covariance
float32[6] acceleration_covariance
float32[6] size_covariance
```